pywinauto/controls/menuwrapper.py

```diff
@@ -71,6 +71,22 @@ class MenuItemNotEnabled(RuntimeError):
     pass
 
 
+class MenuInaccessible(RuntimeError):
+    """
+    Raised when a menu has handle but inaccessible
+    """
+    pass
+
+
+def raise_if_inaccessible(method):

```

Comment:
The code structure looks perfect, but I would suggest having `@ensure_accessible` and `self.accessible` names respectively.